content/millennium/dstu2/care-provision/procedure-request.md

```diff
@@ -0,0 +1,115 @@
+* [Reason](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.reason_x_){:target="_blank"}
+* [Scheduled for](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.scheduled_x_){:target="_blank"}
+* [Patient encounter](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.encounter){:target="_blank"}
+* [Performer](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.performer){:target="_blank"}
+* [Status of the request](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.status){:target="_blank"}
+* [Notes](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.notes){:target="_blank"}
+* [As needed/PRN](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.asNeeded_x_){:target="_blank"}
+* [Ordered date/time](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.orderedOn){:target="_blank"}
+* [Ordered by](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.orderer){:target="_blank"}
+* [Priority](http://hl7.org/fhir/DSTU2/procedurerequest-definitions.html#ProcedureRequest.priority){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:procedure_request, :dstu2) %>
+
+## Search
+
+Search for ProcedureRequests that meet supplied query parameters:
+
+    GET /ProcedureRequest?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name         | Required?                             | Type          | Description
+--------------|---------------------------------------|---------------|------------------------------------------------------------------------------------
+ `_id`        | This, or `patient` or `subject`       | [`token`]     | The logical resource id associated with the ProcedureRequest. Example: `_id=44869605` |
+ `patient`    | This, or `_id` or `subject`           | [`reference`] | The patient associated with the ProcedureRequest. Example: `patient=1316020` or `patient=Patient/1316020` |
+ `subject`    | This, or `_id` or `patient`           | [`reference`] | The subject associated with the ProcedureRequest. Must represent a Patient resource. May use the `:Patient` modifier. Example: `subject=Patient/1316020` or `subject:Patient=1316020`  |
+ `_lastUpdated`  | N                                  | [`date`]      |  An explicit or implied date-time range within which the most recent clinically relevant update was made to the procedure request. Must be prefixed by ‘ge’ or ‘le’. Example: ge2014-05-19T20:54:02.000Z |
+ `_count`       | N                                   | [`number`]    | The maximum number of ProcedureRequests to retrieve in a page. |
+
+Notes:
+
+  - Only one of `patient`, `subject`, and `_id` may be provided
+  - `_count` and `_lastUpdated` will not be allowed to be provided with `_id`
```

Comment:
'may not be provided when `_id` is provided'

Reply:
I was thinking about that..
'may' is less binding than 'should' in the FHIR spec conformance rules. Wouldn't that be confusing? https://www.hl7.org/fhir/conformance-rules.html

Reply:
I suggest 'may not' to match wording already on the site, rather than use "may"'s FHIR keyword definition

L90
https://github.com/cerner/fhir.cerner.com/blob/master/content/millennium/dstu2/care-provision/care-plan.md